### PR TITLE
Sl/cs fixes model folder

### PR DESCRIPTION
**Pull request by simon-letch**

### CODE REVIEW

File: Model/Apiconnector/Client.php [hunk 1]
```diff
@@ -59,8 +60,9 @@ class Client extends \Dotdigitalgroup\Email\Model\Apiconnector\Rest
      * @var \Dotdigitalgroup\Email\Helper\Data
      */
     private $helper;
+
     /**
-     * @var
+     * @var object
      */
     private $apiEndpoint;
```
Comment (cdiacon) [inline Model/Apiconnector/Client.php]: apiendpoint should be string

File: Model/Apiconnector/Client.php [hunk 2]
```diff
@@ -98,11 +100,18 @@ public function __construct(
             self::API_ERROR_ADDRESSBOOK_NOT_FOUND,
         ];
 
+    /**
+     * @param mixed $apiEndpoint
```
Comment (cdiacon) [inline Model/Apiconnector/Client.php]: apiEndpoint = string

File: Model/Apiconnector/Client.php [hunk 3]
```diff
@@ -145,7 +154,7 @@ public function validate($apiUsername, $apiPassword)
     /**
      * Gets a contact by ID. Unsubscribed or suppressed contacts will not be retrieved.
      *
-     * @param $id
+     * @param mixed $id
```
Comment (cdiacon) [inline Model/Apiconnector/Client.php]: id - string

File: Model/Apiconnector/Client.php [hunk 4]
```diff
@@ -361,7 +374,7 @@ public function getAddressBookById($id)
     /**
      * Creates an address book.
      *
-     * @param        $name
+     * @param mixed $name
```
Comment (cdiacon) [inline Model/Apiconnector/Client.php]: string

**Comment by simon-letch** [comment]
@cdiacon 4 points here are sorted

**Pull request merged**